### DISCUSSION TITLE
Should we start documenting LeetCode files?

### DISCUSSION THREAD

**Discussion by Panquesito7** [TheAlgorithms/C#1223] [Ideas]
Currently, all the LeetCode files are not or barely documented, unlike all the other algorithms.
I think it'd be great and very useful if we start documenting these files before we have way too many files and it becomes difficult to manage.

What do you folks think about this? Let me know below.
Thank you for your time. 🙂

**Comment by alexpantyukhin** [comment]
Good idea! I think also we have to keep in mind documentation requirement for new PR's

**Reply by Panquesito7** [reply]
Nice! Would you like to write new guidelines for the same, or should we work on it together? 🙂

**Reply by alexpantyukhin** [reply]
I Like to work together :)

**Reply by Panquesito7** [reply]
Let's discuss it on the Discord, then. :)
